src/appellate/appellate.js

```diff
@@ -0,0 +1,162 @@
+        let result = api_results.results.filter(function (obj) {
+          return obj.pacer_doc_id === pacer_doc_id;
+        })[0];
+        if (!result) {
+          continue;
+        }
+        let href = `https://storage.courtlistener.com/${result.filepath_local}`;
+        let recap_link = $('<a/>', {
+          'class': 'recap-inline',
+          'title': 'Available for free from the RECAP Archive.',
+          'href': href
+        });
+        recap_link.append($('<img/>').attr({
+          src: chrome.extension.getURL('assets/images/icon-16.png')
+        }));
+        recap_link.insertAfter(this.links[i]);
+      }
+      let spinner = document.getElementById("recap-button-spinner")
+      if (spinner){
+          spinner.classList.add('recap-btn-spinner-hidden');
+      }
+    }, this));
+}
+
+
+AppellateDelegate.prototype.handleDocketDisplayPage = async function () {
+  // retrieve pacer_case_id from URL query parameters
+  let pacer_case_id = this.queryParameters.get('caseid') || this.queryParameters.get('caseId')
+
+  // if the last step didn't find the caseId in the query parameter, It will check the storage 
+  if (!pacer_case_id) {
+    const tabStorage = await getItemsFromStorage(this.tabId)
+    if (!tabStorage && !tabStorage.caseId) { return; }
+    pacer_case_id = tabStorage.caseId
+  }
+
+  // Query the first table with case data and insert the RECAP actions button 
+  let table = document.querySelectorAll('table')[3]
+  let button = recapActionsButton(this.court, pacer_case_id, false)
+  table.after(button)
+
+  this.recap.getAvailabilityForDocket(
+    this.court,
+    pacer_case_id,
+    (result) => {
+      if (result.count === 0) {
+        console.warn('RECAP: Zero results found for docket lookup.');
+      } else if (result.count > 1) {
+        console.error(
+          `RECAP: More than one result found for docket lookup. Found ${result.count}`
+        );
+      } else {
+        addAlertButtonInRecapAction(this.court, pacer_case_id)
+        let cl_id = getClIdFromAbsoluteURL(result.results[0].absolute_url)
+        addSearchDocketInRecapAction(cl_id)
+      }
+    }
+  );
+
+  // if you've already uploaded the page, return
+  if (history.state && history.state.uploaded) { return; }
+
+  const options = await getItemsFromStorage('options');
+
+  if (options['recap_enabled']) {
+
+    let callback = (ok) => {
+      if (ok) {
+        addAlertButtonInRecapAction(this.court, pacer_case_id)
+        history.replaceState({ uploaded: true }, '');
+        this.notifier.showUpload(
+          'Docket uploaded to the public RECAP Archive.',
+          function () { }
+        );
+      }
+    };
+
+    this.recap.uploadDocket(this.court, pacer_case_id,
+      document.documentElement.innerHTML,
+      'APPELLATE_DOCKET',
+      (ok) => callback(ok));
+
+    this.attachRecapLinksToEligibleDocs()
+
+  } else {
+    console.info(`RECAP: Not uploading docket. RECAP is disabled.`);
+  }
+}
+
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
You're right. The code review before committing to git makes a big difference in the quality of the work.
 
Thanks for your suggestion. I'll be more careful next time. 

I will check Prettier configuration to avoid these blank lines.